client/src/js/leafletComponents/resultItem.js

```diff
@@ -13,9 +13,11 @@ var localization = require('../localization');
 var resultItem = {
   create: function (options) {
     this._options = options;
+    var colCount = options.isSearchResult ? 'col-xs-9 result'  : 'col-xs-6 c-result-item';
```

Comment:
Now that we are including to css classes, can we rename this variable (`colCount`) to something more a pro pro like `resultItemCss`?